send_message.py

```diff
@@ -33,14 +24,23 @@ def send_message(camera, timestamp, feed_url, img_filename):
 
     msg.attach(image)
 
-    # recipient = phone_number + CARRIERS[carrier]
-    auth = (EMAIL, PASSWORD)
+    email = vault.get_value("sms", 0)
+    password = vault.get_value("sms", 1)
+
+    # Decryption is not needed for carrier addresses since they are not encrypted.
+    carrier_addr = vault.get_env("carriers")[carrier]
+    people = vault.get_env("recipients")
+
+    # Populates list of SMS recipients ["phone_no" + "carrier_addr" ... n]
+    recipients = [vault.get_value(person) + carrier_addr for person in people.keys()] 
```

Comment:
I'm wondering if we should restrict this operation. I think we should have only security.py read from the vault and restrict import to only TFLite_detection_stream.py. This way we can restrict the exposure of data :D

Reply:
I wonder if we can force users to be unable to import security.py outside of TFLite_detection_stream.py? :D

Reply:
@CurranSeam lets re-visit this in another story, but can you create one for this? :D

Reply:
Will do! :D